# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/librsvg/hotfix-armv6.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- librsvg-2.61.1/vendor/cfg-expr/src/targets/builtins.rs.vanilla	2025-10-02 10:53:49.563091910 +0200
+++ librsvg-2.61.1/vendor/cfg-expr/src/targets/builtins.rs	2025-10-02 10:57:49.547292031 +0200
@@ -911,6 +911,32 @@
         panic: Panic::unwind,
     },
     TargetInfo {
+        triple: Triple::new_const("armv6-unknown-linux-gnueabihf"),
+        os: Some(Os::linux),
+        abi: Some(Abi::eabihf),
+        arch: Arch::arm,
+        env: None,
+        vendor: Some(Vendor::unknown),
+        families: Families::unix,
+        pointer_width: 32,
+        endian: Endian::little,
+        has_atomics: HasAtomics::atomic_8_16_32_64_ptr,
+        panic: Panic::unwind,
+    },
+    TargetInfo {
+        triple: Triple::new_const("armv6eb-unknown-linux-gnueabihf"),
+        os: Some(Os::linux),
+        abi: Some(Abi::eabihf),
+        arch: Arch::arm,
+        env: None,
+        vendor: Some(Vendor::unknown),
+        families: Families::unix,
+        pointer_width: 32,
+        endian: Endian::big,
+        has_atomics: HasAtomics::atomic_8_16_32_64_ptr,
+        panic: Panic::unwind,
+    },
+    TargetInfo {
         triple: Triple::new_const("armv7-unknown-linux-gnueabihf"),
         os: Some(Os::linux),
         abi: Some(Abi::eabihf),
